stylesheets/mobile.css

```diff
@@ -0,0 +1,75 @@
+/* CSS for mobile devices */
+
+@media screen and (min-width: 375px) {
+  * {
+    box-sizing: border-box;
+  }
+
+  body {
+    min-width: 375px;
+    font-family: "Poppins", Arial, sans-serif;
+    font-style: normal;
+    max-width: 90vw;
+    margin: 0 auto;
+    height: 674px;
+    background: #fff;
+  }
+
+  .nav-container {
+    height: 48px;
+  }
+
+  .wrapper {
+    max-width: 780px;
+    margin: 0 auto;
+    width: 87.2vw;
+  }
+
+  nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 48px;
+  }
+
+  nav .logo {
+    font-size: 18px;
+    font-weight: 800;
+    color: #6070ff;
+  }
+
+  header {
+    padding-top: 114px;
+  }
+
+  header .content h1 {
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    margin: 0;
+    color: #172b4d;
+  }
+
+  header .content p:first-of-type {
+    color: #344563;
+    font-weight: 800;
+    font-weight: 400;
+    font-size: 16px;
```

Comment:
- [ ] It would be better to make the codes more cleaner and professional by removing unused properties in your codes 😎
 
currently `line 55` and `line 56` are identical 😂